z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -118,6 +118,48 @@ Provision a Red Hat machine based on the following specifications, or using exis
   * openstacksdk >= 0.57.0
   * netaddr
 
+**Install:**
+
+Use the following command to register the system, then automatically associate any available subscription matching that system:
+```sh
+sudo subscription-manager register --username <username> --password <password> --auto-attach
+```
+After registration, use the following command to enable ansible repository.
+```sh
+sudo subscription-manager repos --enable=ansible-2.8-for-rhel-8-s390x-rpms 
+```
+Install the packages from the repository in the system:
+```sh
+sudo yum install python3 ansible jq
```

Comment:
What's the version of ansible after installation?

Reply:
```
[root@lynn-rhel82 yum.repos.d]# ansible --version
ansible 2.8.18
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.8 (default, Dec  5 2019, 15:45:24) [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
```

Reply:
Thanks, but I only tested UPI with Ansible 2.11, have you ever done UPI seccessfully with Ansible 2.8.

Reply:
Hello Xin,
I have tested all changes successfully in ansible 2.8.18.
```
[root@bastion2 ocp_upi]# ansible --version
ansible 2.8.18
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.8 (default, Mar 18 2021, 08:58:30) [GCC 8.4.1 20200928 (Red Hat 8.4.1-1)]
[root@bastion2 ocp_upi]# oc get nodes
NAME                       STATUS   ROLES    AGE    VERSION
host-172-26-105-63         Ready    worker   3m7s   v1.20.0+c8905da
host-172-26-105-68         Ready    worker   3m9s   v1.20.0+c8905da
openshift-5zb9r-master-0   Ready    master   40m    v1.20.0+c8905da
openshift-5zb9r-master-1   Ready    master   40m    v1.20.0+c8905da
openshift-5zb9r-master-2   Ready    master   40m    v1.20.0+c8905da
[root@bastion2 ocp_upi]#
```

Reply:
Verified on ZVM - 163 test stand:
```
[root@os015 ocp_upi]# oc get nodes
NAME                           STATUS   ROLES    AGE    VERSION
openshift-kl6xs-master-0       Ready    master   21m    v1.21.1+051ac4f
openshift-kl6xs-master-1       Ready    master   20m    v1.21.1+051ac4f
openshift-kl6xs-master-2       Ready    master   19m    v1.21.1+051ac4f
openshift-kl6xs-worker-64549   Ready    worker   9m7s   v1.21.1+051ac4f
openshift-kl6xs-worker-f2c77   Ready    worker   10m    v1.21.1+051ac4f
openshift-kl6xs-worker-f49c8   Ready    worker   11m    v1.21.1+051ac4f
[root@os015 ocp_upi]#
```